src/libraries/System.Data.Common/src/System/Data/Common/DBSchemaRow.cs

```diff
@@ -300,8 +300,11 @@ internal bool IsReadOnly
             }
         }
 
+        [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicProperties | DynamicallyAccessedMemberTypes.PublicFields)]
         internal Type? DataType
         {
+            [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2073:UnrecognizedReflectionPattern",
+                Justification = "Type requirements (public properties + public fields) are checked when type is assigned to DataColumn.")]
```

Comment:
Can we verify this somehow? I think at a minimum a TrimmingTest would be justified here.

Reply:
If we can't verify this, I don't think we can add the suppression here, and would need to mark usages of this as `[RUC]`.

Reply:
It looks like the `DataTable` comes from `IDataReader.GetSchemaTable()`.... then that `DataTable` is used to get the DataRow.
Since `IDataReader.GetSchemaTable()` is a public interface, and can be implemented by 3rd parties, I'm not sure we can verify this Type is going to be marked correctly.